gui/slick/views/snatchSelection.mako

```diff
@@ -187,7 +187,7 @@
     <div id="searchNotification"></div>
 
     <div class="clearfix"></div>
-    <div id="wrapper">
+    <div id="wrapper" data-history-toggle="hide">
```

Comment:
This is the meat of the PR. Here I make sure the last status is saved as a data attribute.